### ISSUE TITLE
IsBaseline component of a FactorValueValueObject in api appears to be meaningless

### ISSUE THREAD

**Issue by oganm** [PavlidisLab/Gemma#757]
IsBaseline always appear to be set to false regardless of a factors status as a baseline. Easiest way to see this is the `getDatasetDifferentialExpressionAnalyses` where the `baselineGroup` never has IsBaseline set to true

Example https://dev.gemma.msl.ubc.ca/rest/v2/datasets/2/analyses/differential
![image](https://github.com/PavlidisLab/Gemma/assets/6352379/5dcb69dc-f23b-4e7b-a856-9f12072cf4dd)

**Comment by arteymix** [comment]
Unlike production, the dev server uses MONDO base, so this could be specific to the file we use.

**Comment by oganm** [comment]
No difference between production and dev in this case. And should this even be related to the ontology version?

**Comment by arteymix** [comment]
Sorry about thar, I commented on the wrong issue.

**Comment by arteymix** [comment]
You're absolutely correct, none of our FactorValue are marked as a baseline in the database via `IS_BASELINE`.

**Comment by arteymix** [comment]
The baseline is actually available in the `AnalysisResultSetValueObject`. I think that makes more sense because different analyses can use different baselines.

**Comment by arteymix** [comment]
I'll expose the baseline group in `DifferentialExpressionAnalysisResultSetValueObject` and hide the `isBaseline` field.

**Comment by arteymix** [comment]
It's ready for the 1.30! I might backport it in the 1.29 if you need it now.

**Comment by oganm** [comment]
no need. have a heuristic to determine the baseline that's been working since forever. any further fix can wait for 1.30

**Comment by arteymix** [comment]
Fixed in 5793921e3698b8bc325441590ec9294990faede5 and 90c7443d8ae77a304e32445cf7fb929b63bd1dc0

**Comment by arteymix** [comment]
Good to know!
